.github/workflows/build-and-test-toolchain.yml

```diff
@@ -61,7 +62,7 @@ on:
 
 env:
   BINUTILS_BRANCH: ${{ inputs.binutils_branch || 'woarm64' }}
-  GCC_BRANCH: ${{ inputs.gcc_branch || 'woarm64' }}
+  GCC_BRANCH: ${{ inputs.gcc_branch || 'fix-d' }}
```

Comment:
To be removed once the Windows-on-ARM-Experiments/gcc-woarm64#33 will be merged.